rotkehlchen/accounting/structures/base.py

```diff
@@ -277,6 +280,17 @@ def get_identifier(self) -> str:
     def get_assets(self) -> list[Asset]:
         return [self.asset]
 
+    def has_details(self) -> bool:
+        return self.extra_data is not None and 'sub-swaps' in self.extra_data
```

Comment:
For example the liquity events have `self.extra_data` non empty. If you set the extra condition of `sub-swaps`  the transactions in the liquity page need to handle this scenario too where the flag is false but they do have information in the extra details field. Is this something that you have discussed with the fronted?

Reply:
But the liquity information in extra_data is not shown on the frontend and is needed only for accounting on backend

Reply:
Yes, it is shown in the frontend in the staking section. Not in the evm transactions but yes in the staking page. Is something that has to be known.